crates/tiny/src/config.rs

```diff
@@ -41,7 +43,7 @@ pub(crate) struct Server {
 
     /// Channels to automatically join.
     #[serde(default)]
-    pub(crate) join: Vec<String>,
+    pub(crate) join: Vec<Chan>,
```

Comment:
Can replace with `libtiny_tui`'s `Chan`, if we want to export that type, or keep it like this.